AsyncDisplayKit/TextKit/ASTextKitFontSizeAdjuster.mm

```diff
@@ -104,8 +106,14 @@ - (NSUInteger)lineCountForString:(NSAttributedString *)attributedString
 
 - (CGFloat)scaleFactor
 {
+  if (_measured) {
+    return _scaleFactor;
+  }
+  
   if ([_attributes.pointSizeScaleFactors count] == 0 || isinf(_constrainedSize.width)) {
-    return 1.0;
+    _measured = YES;
```

Comment:
The renderer owns this object, and the text node will create a new renderer when a new string is set (and in other cases as well). I believe this is ok but it is worth more consideration.